libs/blocks/chart/chartLightTheme.js

```diff
@@ -0,0 +1,109 @@
+// Custom Echarts Theme https://echarts.apache.org/en/theme-builder.html
+import { isEmptyObject } from '../../utils/utils.js';
+
+export const LIGHT_SMALL = 'lightSmall';
+export const LIGHT_LARGE = 'lightLarge';
+
+export default (deviceSize) => {
+  if (isEmptyObject(window.echarts)) return null;
```

Comment:
Does this require `isEmptyObject`?  I don't see anywhere that `window.echarts` would be initialized to `{}`
```suggestion
  if (!window.echarts || !Object.keys(window.echarts).length) return null;
```

Reply:
`isEmptyObject` doesn't appear necessary so I removed it. This was pulled in from northstar, so we may have changed things since then